es6.md

```diff
@@ -145,3 +145,49 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ImportDeclaration
+
+**FIXME:** Esprima's behavior differs from Acorn's and SpiderMonkey's behavior in that `source` is of type `ModuleSpecifier`, which is otherwise identical to a `Literal`. See [Esprima bug 1077](https://github.com/jquery/esprima/issues/1077).
+
+```js
+interface ImportDeclaration <: Declaration {
+    specifiers: [ ImportSpecifier ];
+    source: Literal; // Or ModuleSpecifier? (Esprima deviation)
```

Comment:
this is going to be a problem in the future once we introduce a module specifier that is not a literal, and this is suppose to be in ES2016, the front runner at this point is: `import {name} from this module;`, where `this module` is an especial module specifier that can be used to access meta information about the module and the loader. this is the main reason why I've chosen `ModuleSpecifier` for esprima.